### DISCUSSION TITLE
Welcome to Developer_Quiz_Site Discussions!

### DISCUSSION THREAD

**Discussion by jdwilkin4** [freeCodeCamp/Developer_Quiz_Site#112] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AbdelrahmanDwedar** [comment]
Great announcement! 🥳

**Reply by jdwilkin4** [reply]
Hi @mike7866 ! 

If you need help with something for the Developer Quiz Site it would be best to open up a new Github discussion and let us know what you need help with

**Comment by cjpanda** [comment]
This is Wonderful!!! I'm very much interested

**Comment by Yindeish** [comment]
I'm interested too

**Comment by snehpr966** [comment]
Learning by making contributions is what I do with this community.

**Comment by Nene-S** [comment]
Hi my name is Emmanuella, 
I made my first pull request on this repo(about 10 days ago) and it was awesome thanks to @jdwilkin4 .
Learning through contributions is what I do with this community

**Comment by sonicx180** [comment]
Sonicx180 is my name (actually username :D ) and web dev is my game!

**Comment by Abdullahi-abdiaziz** [comment]
I'am interested too!!

**Comment by mehedithedev** [comment]
Hi folks

**Comment by Aman-Singh-Kushwaha** [comment]
Hi @jdwilkin4,
I suggest adding a different category for "Computer Network" with a separate file, which is essential for becoming IT professionals in the Cloud and DevOps field.
What do you say?

**Reply by jdwilkin4** [reply]
Hi @Aman-Singh-Kushwaha ! 

We do have an IT quiz which is where the computer networking questions can go into. 
Plus we do want to keep the categories in line with the Learn to code RPG categories since it is a companion site for that.

**Comment by Rohan-Singhh** [comment]
Hii @jdwilkin4 
https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/715#issue-1922281804
